testutil/testutil.go

```diff
@@ -20,11 +24,19 @@ import (
 	carv2 "github.com/ipld/go-car/v2"
 	"github.com/ipld/go-car/v2/blockstore"
 	"github.com/ipld/go-ipld-prime"
+	"github.com/libp2p/go-libp2p"
 	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
 	"github.com/multiformats/go-multihash"
 	"github.com/stretchr/testify/require"
 )
 
+func NewID(t testing.TB) peer.ID {
+	lp2p, _ := libp2p.New()
```

Comment:
side-note: Never ignore `err` in test utilities specially when we have `t` to assert that no error has occurred.

Reply:
thanks, removed this block